### PR TITLE
Disable response body processing

### LINKED ISSUE #110: Disable buffered response body processing mode
A significant portion of LLM requests are streaming requests, and therefore we can't use buffered processing mode for responses.

/assign @liu-cong

### PR DESCRIPTION
**Pull request by liu-cong**
Fixes #110 

We still keep the HandleResponseBody code so it can be used for testing, or for use cases that don't use streaming.

### CODE REVIEW

File: pkg/manifests/ext_proc.yaml [hunk 1]
```diff
@@ -94,7 +94,6 @@ spec:
         request:
```
Comment (ahg-g) [inline pkg/manifests/ext_proc.yaml:94]: I see we have files in https://github.com/kubernetes-sigs/gateway-api-inference-extension/tree/main/examples/poc/manifests/gateway too, are those still relevant?

Comment (liu-cong) [reply pkg/manifests/ext_proc.yaml:94]: Removed them. They are already outdated.

**Pull request merged**